CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 2.26.0-rc.0 / 2021-03-24
+
+Notable changes:
```

Comment:
The only changes that should stand up are the go 1.16 change and the move to alertmanager api v2.